1. remove unused using, 2. Reorganized the using, 3. Moved the class into separate file and 4. Added line and spacing formats

vishipayyallore opened the pull request.

vishipayyallore commented:
@sughosneo 
Good afternoon. Please look into this whenever you get a couple of minutes. Many thanks.

Regards,
Swamy

sughosneo commented:
Hi @vishipayyallore , thank you for submitting this PR. Will test and update.

sughosneo commented on src/Services/Catalog/Catalog.API/Proto/catalog.proto:
```diff
@@ -1,60 +1,69 @@
-    int32 restock_threshold=12;
-    int32 max_stock_threshold=13;
-    bool on_reorder=14;
+    
```
I would personally just avoid these extra line break.

sughosneo commented on src/Services/Catalog/Catalog.API/IntegrationEvents/Events/OrderStockItem.cs:
```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.eShopOnContainers.Services.Catalog.API.IntegrationEvents.Events
```
I wouldn't recommend to create a separate class file for class `OrderStockItem` for the following reason. 
- It's specific to `OrderStockConfirmedIntegrationEvent`
- And the `Events` directory contains all events. Keeping a public class as a separate file might not make much sense.

vishipayyallore replied:
@sughosneo 
Sure let me address those and ping you. Many thanks.

sughosneo commented:
Hi @vishipayyallore - I have included few suggestions. If you could review those. 

Thank you.

vishipayyallore commented:
> Hi @vishipayyallore - I have included few suggestions. If you could review those.
> 
> Thank you.

@sughosneo 
I have reverted the commit. I will exclude those two suggestions and I will check in the code. Many thanks.

vishipayyallore commented:
> > Hi @vishipayyallore - I have included few suggestions. If you could review those.
> > Thank you.
> 
> @sughosneo
> I have reverted the commit. I will exclude those two suggestions and I will check in the code. Many thanks.

@sughosneo (Sumit),
Good afternoon. I have addressed those comments. Could you please review it? I will create another PR after this PR gets merged. Many thanks.

Regards,
Swamy

sughosneo commented:
- Reviewed the changes for the following project 
`Basket.API`,`Catalog.API`,`Webhooks.API`,`Application.FunctionalTests`
- Tested the changes with `docker-compose`. LGTM
